### ISSUE TITLE
Override config file

### ISSUE THREAD

**Issue by bergercookie** [tealdeer-rs/tealdeer#107]
Hi, thanks for this tool!

It would be really useful to add a flag for the user to specify a custom configuration file, something like `--override-config <...>`

In my case, I want to call `tldr` from [albert](https://albertlauncher.github.io/) and in that case (only in that case, not system-wide) I 'd like to specify a custom configuration file which overrides the `use_pager` option

**Comment by dbrgn** [comment]
Hi! Could you maybe override `XDG_CONFIG_HOME`?

```
$ XDG_CONFIG_HOME=/tmp tldr --config-path
Config path is: /tmp/tealdeer/config.toml
```

**Comment by bergercookie** [comment]
That could be a way I suppose, but it's more of a hack rather than a solution to this.

This isn't super important to me, I can bypass it and parse the markdown files directly. I raised the issue because I thought it was a nice feature to have.

FWIW, here's the albert plugin I'm using `tldr` for: https://github.com/bergercookie/awesome-albert-plugins/tree/master/plugins/tldr_pages

**Comment by dbrgn** [comment]
Yeah, I suppose a `--config` flag wouldn't hurt :slightly_smiling_face:

**Comment by niklasmohrin** [comment]
This could also be considered when running with the `--seed-config` flag to specify where to put the generated file. I think the option should then have a more verbose name like `config-location` (unfortunately `config-path` is already taken).

**Comment by dmaahs2017** [comment]
> (unfortunately `config-path` is already taken)

Should `--config-path` be deprecated considering we have `--show-paths`? Or perhaps `--config-path` should be renamed to `--show-config-path`.

Or for consistency, deprecate `--config-path` in preference for something like this `--show-path <config|cache|pages|custom>`. Then we would have `show-paths` and `show-path <enum>` which would be a bit more intuitive.

Isn't so relevant to _this_ issue, but is this worth raising a new issue for @dbrgn @niklasmohrin ?

**Comment by dbrgn** [comment]
`--config-path` is already deprecated:

```
danilo@c3po:~$ tldr --config-path
Warning: The --config-path flag is deprecated, use --show-paths instead
Config path is: /home/danilo/.config/tealdeer/config.toml
```

It will be removed in a future version.

**Comment by niklasmohrin** [comment]
About `--show-path <config|cache|pages|custom>`: I feel like `--show-paths` is primarily intended for humans to understand how path precedence works with tealdeer or to initially find a place of interest. Having the single `--show-path` to me seems like a convenience thing for scripts, but they can just _know_ the location or grep it from `--show-paths`. Unless specific feature requests come in, I would not bother to implement it
